Firestore/core/src/firebase/firestore/util/path.cc

```diff
@@ -16,10 +16,41 @@
 
 #include "Firestore/core/src/firebase/firestore/util/path.h"
 
+#include "absl/strings/ascii.h"
+#include "absl/strings/string_view.h"
+
 namespace firebase {
 namespace firestore {
 namespace util {
 
+namespace {
+
+#if defined(_WIN32)  // not yet used elsewhere
+static constexpr absl::string_view::size_type npos = absl::string_view::npos;
+#endif
+
+/** Returns the given path with its leading drive letter removed. */
+inline absl::string_view StripDriveLetter(absl::string_view path) {
+#if defined(_WIN32)
+  if (path.size() >= 2 && path[1] == ':' && absl::ascii_isalpha(path[0])) {
+    return path.substr(2);
+  }
+#endif  // defined(_WIN32)
```

Comment:
Ultranit: have the default implementation in an `#else` clause for consistency with `IsSeparator`?

Reply:
Not sure what you're asking for. `return path;` applies in both cases. Are you asking for this?

```c++
inline absl::string_view StripDriveLetter(absl::string_view path) {
#if defined(_WIN32)
  if (path.size() >= 2 && path[1] == ':' && absl::ascii_isalpha(path[0])) {
    return path.substr(2);
  }
  return path;

#else
  return path;
#endif  // defined(_WIN32)
}
```

Reply:
Yes, that's what I meant. I'm totally fine if you leave it as is, sorry about being confusing.

Reply:
I have no strong opinion one way or the other, but since you mentioned it. Done.